From 5c2c7b38f13d0313a4c75e563e65ff961fb67d9f Mon Sep 17 00:00:00 2001
From: Mislav Novakovic <mislav.novakovic@sartura.hr>
Date: Mon, 12 Dec 2016 18:08:00 +0100
Subject: [PATCH] in cmake remove section 'install server configuration module
 and enable features'

Signed-off-by: Mislav Novakovic <mislav.novakovic@sartura.hr>
---
 server/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/server/CMakeLists.txt b/server/CMakeLists.txt
index ea2035d..1c87940 100644
--- a/server/CMakeLists.txt
+++ b/server/CMakeLists.txt
@@ -124,6 +124,7 @@ endif()
 # install binary
 install(TARGETS netopeer2-server DESTINATION ${CMAKE_INSTALL_BINDIR})
 
+if(FALSE)
 # install server configuration module and enable features
 install(CODE "
     execute_process(COMMAND ${SYSREPOCTL_EXECUTABLE} -l RESULT_VARIABLE RET OUTPUT_VARIABLE INSTALLED_MODULES ERROR_VARIABLE OUT)
@@ -203,6 +204,7 @@ install(CODE "
         endif()
 
     endif()")
+endif()
 
 # clean cmake cache
 add_custom_target(cleancache
-- 
2.10.2

